config/samples/konnect_kongvault.yaml

```diff
@@ -0,0 +1,38 @@
+kind: KonnectAPIAuthConfiguration
+apiVersion: konnect.konghq.com/v1alpha1
+metadata:
+  name: konnect-api-auth-dev-1
+  namespace: default
+spec:
+  type: token
+  token: kpat_XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+  serverURL: us.api.konghq.com
+---
+kind: KonnectGatewayControlPlane
+apiVersion: konnect.konghq.com/v1alpha1
+metadata:
+  name: test1
+  namespace: default
+spec:
+  name: test1
+  labels:
+    app: test1
+    key1: test1
+  konnect:
+    authRef:
+      name: konnect-api-auth-dev-1
+---
+kind: KongVault
+apiVersion: configuration.konghq.com/v1alpha1
+metadata:
+  name: env-vault-1
+spec:
+  backend: env
+  prefix: env-vault
+  config:
+    prefix: "konnect_vault_test_"
+  controlPlaneRef:
+    type: konnectNamespacedRef
+    konnectNamespacedRef:
+      name: test1
+      namespace: default
```

Comment:
Can we add a comment here that the namespace is specified because `KongVault` is cluster scoped currently?

Reply:
Done.